firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/Serial/Nonin/PacketReceiver.h

```diff
@@ -103,39 +53,22 @@ inline uint16_t get_6bit_data(uint8_t byte_data) {
  * @param  lsbByte - MSB to extract 5 bits
  * @return Calculated 9 bit data from MSB and LSB
  */
-inline uint16_t get_9bit_data(uint8_t msb_byte, uint8_t lsb_byte) {
+inline uint16_t get_hr_data(uint8_t msb_byte, uint8_t lsb_byte) {
   /* Pack 2 bits of MSB and 6 bits of LSB for 9 bits of heart rate data  */
   static const uint16_t msb_shift = 7;
   static const uint16_t mask_msb = 0x18;
-  static const uint16_t mask_9bit = 0x01FF;
+  static const uint16_t hr_missing_data = 0x01FF;
```

Comment:
If `hr_missing_data` is actually a value defined in the datasheet, perhaps we should move into Types.h together with the "SpO2 missing" flag